docs/general/kusama/kusama-parameters.md

```diff
@@ -71,8 +71,8 @@ The Kusama Council consists of up to 19 members and up to 19 runners up.
 ### Staking, Validating, and Nominating
 
 A maximum of
-{{ polkadot: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :polkadot }}
-{{ kusama: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :kusama }}
+{{ polkadot: <RPC network="polkadot" path="staking.nominationsquota" defaultValue={16}/> :polkadot }}
```

Comment:
I think Keith advised using both polkadot and kusama conditional renders even if we are just talking about one network. This page is kusama parameters, so in both Polkadot Wiki and Kusama Guide, this should just show kusama values.

```suggestion
{{ polkadot: <RPC network="kusama" path="staking.nominationsquota" defaultValue={24}/> :polkadot }}
```